lib/src/main/java/net/cachapa/expandablelayout/ExpandableLinearLayout.java

```diff
@@ -211,7 +223,7 @@ public void setOnExpansionUpdateListener(OnExpansionUpdateListener listener) {
         this.listener = listener;
     }
 
-    private void animateHeight(final View view, final int targetHeight) {
+    private void animateHeight(final View view, final int targetHeight,boolean animated) {
```

Comment:
I'm not a fan of return statements in the middle of methods, as it makes it hard to follow the code path.
The non-animated option should be separated into its own function.